README.md

```diff
@@ -314,6 +315,27 @@ public final class App {
 }
 ```
 
+## Front interface
+
+Essential part of [Bigger Example](#a-bigger-example) is 
+[Front](src/main/java/org/takes/http/Front.java) interface.
+It is representing an HTTP front that will start the server,
```

Comment:
@t-izbassar actually this `Front` can start non-http servers too ;) I'd just say "front" - is an object which encapsulates server's "back" and can be started.